Client/Frontend/TabContentsScripts/FormAutofillHelper/FormAutofillHelper.swift

```diff
@@ -136,44 +112,36 @@ class CreditCardHelper: TabContentScript {
               card.ccExpYear > 0,
               !card.ccName.isEmpty
         else {
-            completion(CreditCardHelperError.injectionInvalidFields)
+            completion(FormAutofillHelperError.injectionInvalidFields)
             return
         }
 
         do {
-            let jsonBuilder = CreditCardHelper.injectionJSONBuilder(card: card)
+            let jsonBuilder = FormAutofillHelper.injectionJSONBuilder(card: card)
             let jsonData = try JSONSerialization.data(withJSONObject: jsonBuilder)
             guard let jsonDataVal = String(data: jsonData, encoding: .utf8) else {
-                completion(CreditCardHelperError.injectionInvalidJSON)
+                completion(FormAutofillHelperError.injectionInvalidJSON)
                 return
             }
 
             guard let webView = tab.webView else {
-                completion(CreditCardHelperError.injectionIssue)
+                completion(FormAutofillHelperError.injectionIssue)
                 return
             }
 
             let fillCreditCardInfoCallback = "__firefox__.FormAutofillHelper.fillFormFields(\(jsonDataVal))"
             webView.evaluateJavascriptInDefaultContentWorld(fillCreditCardInfoCallback, frame) { _, error in
-                guard let error = error else {
-                    TelemetryWrapper.recordEvent(category: .action,
-                                                 method: .tap,
-                                                 object: .creditCardAutofilled)
+                if let error = error {
+                    TelemetryWrapper.recordEvent(category: .action, method: .tap, object: .creditCardAutofillFailed)
+                    completion(error)
+                    logger.log("Credit card script error \(error)", level: .debug, category: .webview)
+                } else {
+                    TelemetryWrapper.recordEvent(category: .action, method: .tap, object: .creditCardAutofilled)
```

Comment:
⚠️  nit: indent
```suggestion
        if let error = error {
            TelemetryWrapper.recordEvent(category: .action,
                                         method: .tap,
                                         object: .creditCardAutofillFailed)
            completion(error)
            logger.log("Credit card script error \(error)",
                       level: .debug,
                       category: .webview)
        } else {
            TelemetryWrapper.recordEvent(category: .action,
                                         method: .tap,
                                         object: .creditCardAutofilled)
```

Reply:
some other places the indentation seemed off but just showing you the two areas for example @jnrahme, easy to update :)

Reply:
Done